examples/text/text_input_box.c

```diff
@@ -113,15 +112,3 @@ int main(void)
 
     return 0;
 }
-
```

Comment:
Please, do not remove this function, it could be useful as a reference for other users.